v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -209,5 +213,47 @@ def requires(self) -> ReferenceDataset:
             RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/4.0/pca/gnomad.v4.0.pca_loadings.ht',
         },
     },
+    ReferenceDataset.gnomad_mito: {
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.MITO]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://gcp-public-data--gnomad/release/3.1/ht/genomes/gnomad.genomes.v3.1.sites.chrM.ht',
+        },
+    },
+    ReferenceDataset.gnomad_non_coding_constraint: {
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://seqr-reference-data/GRCh38/gnomad_nc_constraint/gnomad_non-coding_constraint_z_scores.ht',
+        },
+    },
+    ReferenceDataset.screen: {
+        ENUMS: {
+            'region_type': [
+                'CTCF-bound',
+                'CTCF-only',
+                'DNase-H3K4me3',
+                'PLS',
+                'dELS',
+                'pELS',
+                'DNase-only',
+                'low-DNase',
+            ],
+        },
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: 'gs://seqr-reference-data/GRCh38/ccREs/GRCh38-ccREs.ht',
+        },
+    },
+    ReferenceDataset.local_constraint_mito: {
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.MITO]),
+            VERSION: '1.0',
+            # Originally sourced from https://www.biorxiv.org/content/10.1101/2022.12.16.520778v2.supplementary-material
+            # Supplementary Table 7.
+            RAW_DATASET_PATH: 'gs://seqr-reference-data/GRCh38/mitochondrial/local_constraint.tsv',
```

Comment:
Same as above. In this case I think we had downloaded it because we did not want to deal with zip archives which is how this table is available in biorxiv but now that we wrote helpers for downloading tables from zip files we should use the original host instead: 
https://www.biorxiv.org/content/biorxiv/early/2023/01/27/2022.12.16.520778/DC3/embed/media-3.zip?download=true

Reply:
this is my last todo on this PR